resources/keycloak/custom-user-provider/src/main/java/com/cvmanager/auth/provider/user/pojos/OrganizationObject.java

```diff
@@ -0,0 +1,96 @@
+package com.cvmanager.auth.provider.user.pojos;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize
+public class OrganizationObject {
+    /*
+     * This class is used to represent a user's organization and role association object from the user_organization postgresql table
+     */
+
+    private static final Logger log = LoggerFactory.getLogger(OrganizationObject.class);
+    private static ObjectMapper objectMapper = new ObjectMapper();
+
+    @JsonProperty("org")
+    private String org;
+    @JsonProperty("role")
+    private String role;
+
+    public static List<OrganizationObject> listFromString(String json) {
+        try {
+            if (json == null || json.isEmpty()) {
+                return null;
```

Comment:
to protect against `NullPointerExceptions`, we can pass back an empty list instead
```suggestion
                return Collections.emptyList();
```

Reply:
updated